<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCutDialog</class>
 <widget class="QDialog" name="QCutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图像裁剪</string>
  </property>
  <property name="windowIcon">
   <iconset resource="picdealsrc.qrc">
    <normaloff>:/resources/PicDeal.ico</normaloff>:/resources/PicDeal.ico</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>左上角坐标</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LT_X">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>，</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LT_Y">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="RB_Y">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>，</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>右下角坐标</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RB_X">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BNOK">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BNCAL">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="picdealsrc.qrc"/>
 </resources>
 <connections/>
</ui>
